src/main/java/org/junit/internal/builders/AllDefaultPossibilitiesBuilder.java

```diff
@@ -40,7 +40,10 @@ protected JUnit3Builder junit3Builder() {
     }
 
     protected AnnotatedBuilder annotatedBuilder() {
-        return new AnnotatedBuilder(this);
+        // For compatibility with Request#classes(...) and the Suite runner,
+        // deeper nesting levels always need to support suite() methods:
+        RunnerBuilder suiteBuilder = canUseSuiteMethod ? this : new AllDefaultPossibilitiesBuilder(true);
```

Comment:
As far as I can tell, the only way `canUseSuiteMethod` can be false is if you either explicitly created `AllDefaultPossibilitiesBuilder` and passed in false, or if you used `Request.classWithoutSuiteMethod()`. In either case, not using the suite method sounds like expected behavior.

Am I missing something